runner/docker/Dockerfile.live-base-comfyui

```diff
@@ -35,14 +35,14 @@ RUN cd /comfyui/custom_nodes && \
     cd ComfyUI-Misc-Effects && \
     git checkout c6b360c78611134c3723388170475eb4898ff6b7
 
-RUN pip install torch==2.5.1 torchvision torchaudio tqdm nvidia-ml-py==12.560.30
```

Comment:
@leszko The only scenario where hardcoding this in the Dockerfile seems necessary is when a dependency chain installs an incorrect version of `nvidia-ml-py` before the `requirements.txt` file is processed. If this is the case we should remove it from the requirements so that we don't have two definitions that can conflict. Can you check that it works without hardcoding it in the docker?